pom.xml

```diff
@@ -68,6 +68,21 @@
   </developers>
 
   <profiles>
+    <!-- JDK10 -->
+    <profile>
+      <id>java10</id>
+      <properties>
+        <!-- Not use alpn agent as Java10 supports alpn out of the box -->
+        <argLine.alpnAgent />
+        <forbiddenapis.skip>true</forbiddenapis.skip>
+        <!-- Needed because of https://issues.apache.org/jira/browse/MENFORCER-275 -->
+        <enforcer.plugin.version>3.0.0-M1</enforcer.plugin.version>
+      </properties>
+      <activation>
+        <jdk>10</jdk>
+      </activation>
```

Comment:
nit: probably better moving this section right after `<id/>` for readability?

Reply:
sure why not